buildout.cfg

```diff
@@ -61,7 +61,7 @@ webtest = git https://github.com/Pylons/webtest.git
 WSGIProxy2 = git https://github.com/lrowe/WSGIProxy2.git
 zope.sqlalchemy = git https://github.com/zopefoundation/zope.sqlalchemy.git
 pytest-bdd = git https://github.com/lrowe/pytest-bdd.git branch=allow-any-step-order
-snovault = git https://github.com/ENCODE-DCC/snovault.git rev=1.0.26
+snovault = git https://github.com/ENCODE-DCC/snovault.git rev= SNO-85-facet-reappearing
```

Comment:
I think this should be “branch=SNO-85-facet-reappearing.” It didn’t build locally until I changed that, though I guess long term it doesn’t matter.

Reply:
yea, this is not the correct format.  @zoldello Please look at 

https://github.com/ENCODE-DCC/encoded/pull/2667/files

for correct format.

Reply:
Fixed.